packages/neo-one-smart-contract-lib/src/ownership/Ownable.ts

```diff
@@ -0,0 +1,52 @@
+import { Address, constant, createEventNotifier, SmartContract } from '@neo-one/smart-contract';
+
+/**
+ * @title Ownable
+ * @dev A ownable contract has an owner address
+ */
+
+/* tslint:disable-next-line: variable-name */
+const transferred_ownership = createEventNotifier<Address | undefined, Address>('transferred ownership', 'from', 'to');
```

Comment:
This should be something like

`const transferOwner = createEventNotifier<Address | undefined, Address>('transfer_owner', 'from', 'to');`

The variable name should follow the same rules as any other variable (camelcase) and the event should always be snake case.

Reply:
@dicarlo2  Thanks for spelling it out this time.  Last time I think I was bit by a thing that @afragapane and I observed: this scroll-bar hiding what you were commenting on
![image](https://user-images.githubusercontent.com/39564353/55901932-3df5bd80-5b7f-11e9-9769-4234d28be07b.png)
https://github.com/neo-one-suite/neo-one/pull/1057